.circleci/config.yml

```diff
@@ -1,13 +1,59 @@
-# Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
+---
+
 version: 2.1
-# Use a package of configuration called an orb.
-orbs:
-  # Declare a dependency on the welcome-orb
-  welcome: circleci/welcome-orb@0.4.1
-# Orchestrate or schedule a set of jobs
+
+jobs:
+  ci:
+    # ref: https://circleci.com/docs/2.0/executor-types/?section=executors-and-images#using-docker
+    docker:
+      # Use the base circle CI image
+      - image: "cimg/base:stable"
+        auth:
+          username: DOCKERHUB_LOGIN
+          password: DOCKERHUB_PASSWORD
+    steps:
+      # TODO set up source and pip caching
+      # checkout the git repo
+      - checkout
+
+      - run:
+          name: "Run tests and code coverage"
+          command: |
+            pip install -U pip wheel
+            pip install poetry
+            poetry install
+            poetry run python groclient/lib.py -v
+            mkdir -p shippable/testresults shippable/codecoverage &&
+            poetry run pytest
+            --junitxml=shippable/testresults/nosetests.xml
+            --cov=groclient
+            --cov-report=xml:shippable/codecoverage/coverage.xml
```

Comment:
Let's remove --junitxml and --cov-report, since those seem tailored to shippable?